examples/features/csm_observability/README.md

```diff
@@ -0,0 +1,30 @@
+# CSM Observability
+
+This examples shows how to configure CSM Observability for a binary, and shows
+what type of telemetry data it can produce for certain RPC's with additional CSM
+Labels. The Client accepts configuration from an xDS Control plane as the
+default address that it connects to is "xds:///helloworld:50051", but this can
```

Comment:
This line is not very clear to me:

```
The Client accepts configuration from an xDS Control plane as the
default address that it connects to is "xds:///helloworld:50051", but this can
be overridden with the command line flag --addr.
```

Is this line even important at the top-level readme file?

Reply:
Who is responsible for running the xDS control plane? How are the resources configured on it? What configuration does the client expect to receive here?

Is this the gRPC client or the client application?

Reply:
This is the language used to describe in C++: https://github.com/grpc/grpc/blob/b18bd9564a7a63a2d180341f5183643525d51515/examples/cpp/csm/observability/README.md#configuration. I called this out because our other examples work by default locally, but this one does not (needs the full xDS System).

Reply:
Specifically the gRPC Client. The xDS Control Plane management/ xDS Configuration is part of the deployment outlined in the in draft user guide. I added "This can be plugged into the steps outlined in the CSM Observability User Guide." to the end of this section. Let me know if you think this is sufficient. Once the user guide goes public I can add a hyperlink.

Reply:
@yashykt

Reply:
There's a bit of a chicken and egg problem here since the user guide for CSM with grpc has not been published yet. 

The example does not make sense without the user guide, but the user guide also needs a working example before it can be published.